Src/GoogleApis/Apis/Requests/BatchRequest.cs

```diff
@@ -254,8 +254,13 @@ internal static HttpResponseMessage ParseAsHttpResponse(string content)
                 while (!string.IsNullOrEmpty((line = reader.ReadLine())))
                 {
                     var separatorIndex = line.IndexOf(':');
-                    headersDic.Add(line.Substring(0, separatorIndex).Trim(),
-                        line.Substring(separatorIndex + 1).Trim());
+                    var key = line.Substring(0, separatorIndex).Trim();

```

Comment:
[optional] I would even add a comment that the following lines fixes https://github.com/google/google-api-dotnet-client/issues/548, batch response with duplicate header